src/style.css

```diff
@@ -0,0 +1,139 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 400;
+}
+
+body {
+  background-color: rgb(225, 227, 229);
+}
+  
+a,
+p,
+body,
+div,
+header,
+html,
+img,
+li,
+nav,
+ul,
+h3 {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  color: #060b35;
+}
+  
+.card__image {
+  width: 160px;
+  height: 134px;
+}
+  
+.card__product-description {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  margin-top: 40px;
+}
+  
+.card__product-title {
+  font-weight: 500;
+  line-height: 18px;
+}
+  
+.card__product-code {
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin-top: 4px;
+}
+  
+.card__review {
+  margin-top: 16px;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+  
+.card__review-count {
+  font-size: 10px;
+  line-height: 14px;
+}
+  
+.card__price {
+  margin-top: 24px;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+  
+.card__price-label {
+  line-height: 18px;
+  color: #616070;
+}
+  
+.card__price-amount {
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+}
+  
+.card__buy-button {
+  margin-top: 16px;
+  text-decoration: none;
+  text-transform: uppercase;
+  text-align: center;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 40px;
+  width: 100%;
+  height: 40px;
+  color: #fff;
+  background-color: #00acdc;
+  border: 1px solid #00acdc;
+  border-radius: 5px;
+  transition: 0.5s;
```

Comment:
list all transition properties, it is better for performance. and move 0.5s to a variable